src/client/components/pages/Courses/CoursesPage.tsx

```diff
@@ -322,6 +339,34 @@ const CoursesPage: FunctionComponent = (): ReactElement => {
     });
   }, [modalButtonId, setNotesModalData]);
 
+  /**
+   * Takes the specified course and displays a modal to edit course enrollment
+   */
+  const openEnrollmentModal = useCallback(
+    (course: CourseInstanceResponseDTO, term: TERM) => {
+      setEnrollmentModalData({
+        course,
+        visible: true,
+        term: term.toUpperCase() as TERM,
```

Comment:
Makes sense. I also had to remove the `.toUpperCase()` to avoid TS saying `Type 'string' is not assignable to type 'TERM'.ts(2322)`.

Fixed in 517bf194